addon.xml

```diff
@@ -10,7 +10,7 @@
         <import addon="script.module.simplejson" version="3.16.1"/>
     </requires>
     <extension point="xbmc.python.pluginsource" library="default.py">
-        <provides>executable</provides>
+        <provides>game executable</provides>
```

Comment:
The addon should provide `game`.

> The extension has an additional child element named <provides>, which contains a whitespace separated list of image, video, audio, and/or executable. This determines in what area (or context) of the Kodi system your addon will make itself visible in (please note that this applies only to plugin extension points):

If both `executable`and `game` is included in the `<provides>` tag the addon shows up in both “Games” and “Programs” menu in Kodi. However, the addon will not construct a list item if called from “Programs” if `xbmcplugin.setContent(handle, content)` is set to `games`

Reply:
Ok I will change that!